src/plugins/core/configuration/plugin-configuration.ts

```diff
@@ -124,7 +124,7 @@ export class PluginConfigurationData {
 	}
 
 	logo() {
-		const logo: string | undefined = this.#config.get<string>("logo");
+		const logo: string | undefined = this.#config.get<Record<string, string>>("desktop-wallet")?.logo;
```

Comment:
@luciorubeens @faustbrian can you double-check that this path for is correct?

Reply:
`this.#config.get<string>("logo");` is the correct one